### PR TITLE
Update onboarding.md

### PR DESCRIPTION
**Pull request by hellotaran**
Updated the proper navigation. Currently we are missing "SALES" which makes user to go through each tab to look for "Checkout" menu.

### CODE REVIEW

File: help/express-checkout/onboarding.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ To connect your payment services provider follow the steps described in the [pro
 
 ## Enable extension
 
-1. On the _Admin_ sidebar, navigate to **Stores** > **Configuration** > **Checkout** to access the Checkout Admin configuration page.
+1. On the _Admin_ sidebar, navigate to **Stores** > **Configuration** > **SALES** > **Checkout** to access the Checkout Admin configuration page.
```
Comment (leslietilling) [inline help/express-checkout/onboarding.md:67]: This is really a two-step thing and not one long menu path:

```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
1. In the left panel, expand **Sales** and choose **Checkout**.
```
@danidelcar  I will let you confirm that this is the correct navigation.

**Comment by hellotaran** [comment]
Merged the changes received from @leslietilling

**Review by danidelcar — APPROVED** [review]
good catch!

**Comment by danidelcar** [comment]
PR in private repo created to import changes

**Pull request closed without merging**